apps/meteor/tests/e2e/e2e-encryption.spec.ts

```diff
@@ -173,6 +176,8 @@ test.describe.serial('e2e-encryption', () => {
 
 		await poHomeChannel.content.sendMessage('hello world not encrypted');
 
+		await page.waitForTimeout(10000);
+
```

Comment:
Why is this timeout so high?

Reply:
It isn't supposed to be there, I was testing something and forgot to remove...oops. 😬 

Will remove it